osbuild/util/bls.py

```diff
@@ -0,0 +1,35 @@
+import glob
+import os
+
+"""
```

Comment:
I see now that this is all just moved code, sorry! Fine of course to keep for now but I would love to have a followup with some tweaks here (I can offer to do that of course)